### PR TITLE
[FLINK-17581][docs-zh] Update translation of S3 documentation

### PR DESCRIPTION
**Pull request by wydhcws**
## What is the purpose of the change

Update translation of S3 Chinese documentation

The change in https://github.com/apache/flink/commit/7c5ac3584e42a0e7ebc5e78c532887bf4d383d9d needs to be added to the Chinese variant of the documentation page.

## Brief change log

  - Update translation of S3 Chinese documentation about  https://github.com/apache/flink/commit/7c5ac3584e42a0e7ebc5e78c532887bf4d383d9d

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`:  no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

File: docs/ops/filesystems/s3.zh.md [hunk 1]
```diff
@@ -58,11 +58,15 @@ env.setStateBackend(new FsStateBackend("s3://<your-bucket>/<endpoint>"));
 Flink 提供两种文件系统用来与 S3 交互：`flink-s3-fs-presto` 和 `flink-s3-fs-hadoop`。两种实现都是独立的且没有依赖项，因此使用时无需将 Hadoop 添加至 classpath。
 
   - `flink-s3-fs-presto`，通过 *s3://* 和 *s3p://* 两种 scheme 使用，基于 [Presto project](https://prestodb.io/)。
-  可以使用与[配置 Presto 文件系统](https://prestodb.io/docs/0.187/connector/hive.html#amazon-s3-configuration)相同的方法进行配置，即将配置添加到 `flink-conf.yaml` 文件中。推荐使用 Presto 文件系统来在 S3 中建立 checkpoint。
+  可以使用[和 Presto 文件系统相同的配置项](https://prestodb.io/docs/0.187/connector/hive.html#amazon-s3-configuration)进行配置，方式为将配置添加到 `flink-conf.yaml` 文件中。如果要在 S3 中建立 checkpoint，推荐使用 Presto S3 文件系统。
```
Comment (klion26) [inline docs/ops/filesystems/s3.zh.md:61]: `如果要在 S3 中建立 checkpoint，推荐使用 Presto S3 文件系统。` 建议改成 `如果要在 S3 中使用 checkpoint，推荐使用 Presto S3 文件系统。`

File: docs/ops/filesystems/s3.zh.md [hunk 2]
```diff
@@ -58,11 +58,15 @@ env.setStateBackend(new FsStateBackend("s3://<your-bucket>/<endpoint>"));
...
 
   - `flink-s3-fs-hadoop`，通过 *s3://* 和 *s3a://* 两种 scheme 使用, 基于 [Hadoop Project](https://hadoop.apache.org/)。
-  文件系统可以使用与 [Hadoop S3A 完全相同的配置方法](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#S3A)进行配置，即将配置添加到 `flink-conf.yaml` 文件中。它是唯一一个支持 [StreamingFileSink]({{ site.baseurl}}/zh/dev/connectors/streamfile_sink.html) 的文件系统。
-
+  本文件系统可以使用类似 [Hadoop S3A 的配置项](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#S3A)进行配置，方式为将配置添加到 `flink-conf.yaml` 文件中。
+  
+     例如, Hadoop 有 `fs.s3a.connection.maximum` 的配置选项. 如果你想在 Flink 程序中改变该配置的值, 你需要将配置 `s3.connection.maximum: xyz` 添加到 `flink-conf.yaml` 文件中. Flink 会内部将其转换成配置 `fs.s3a.connection.maximum`. 而无需通过 Hadoop 的XML 配置文件来传递参数.
```
Comment (klion26) [inline docs/ops/filesystems/s3.zh.md:66]: 这里建议都使用中文标点符号

File: docs/ops/filesystems/s3.zh.md [hunk 3]
```diff
@@ -58,11 +58,15 @@ env.setStateBackend(new FsStateBackend("s3://<your-bucket>/<endpoint>"));
...
+  
+    另外，它是唯一支持 [StreamingFileSink]({{ site.baseurl}}/dev/connectors/streamfile_sink.html)的S3 文件系统.
```
Comment (klion26) [inline docs/ops/filesystems/s3.zh.md:68]:
```suggestion
    另外，它是唯一支持 [StreamingFileSink]({{ site.baseurl}}/zh/dev/connectors/streamfile_sink.html)的S3 文件系统.
```

Comment (klion26) [reply docs/ops/filesystems/s3.zh.md:68]: 关于链接你可以在 [wiki](https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications?src=contextnavpagetreemode) 中查看更多相关说明

File: docs/ops/filesystems/s3.zh.md (hunk 1 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: 谢谢指出问题 已经修改

File: docs/ops/filesystems/s3.zh.md (hunk 2 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: 谢谢指出问题 已经修改

File: docs/ops/filesystems/s3.zh.md (hunk 3 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: 谢谢指出问题 已经修改

File: docs/ops/filesystems/s3.zh.md (hunk 2 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: > 这里建议都使用中文标点符号



> 这里建议都使用中文标点符号

谢谢指出问题 已经修改

Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: > 这里建议都使用中文标点符号
谢谢指出问题 已经修改

Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: > 这里建议都使用中文标点符号

谢谢指出问题 已经修改

**Comment by wydhcws** [comment]
Thanks for your suggestions. I've pushed a new Commit.  @klion26

File: docs/ops/filesystems/s3.zh.md [hunk 4]
```diff
@@ -58,11 +58,15 @@ env.setStateBackend(new FsStateBackend("s3://<your-bucket>/<endpoint>"));
...
   - `flink-s3-fs-presto`，通过 *s3://* 和 *s3p://* 两种 scheme 使用，基于 [Presto project](https://prestodb.io/)。
-  可以使用与[配置 Presto 文件系统](https://prestodb.io/docs/0.187/connector/hive.html#amazon-s3-configuration)相同的方法进行配置，即将配置添加到 `flink-conf.yaml` 文件中。推荐使用 Presto 文件系统来在 S3 中建立 checkpoint。
+  可以使用[和 Presto 文件系统相同的配置项](https://prestodb.io/docs/0.187/connector/hive.html#amazon-s3-configuration)进行配置，方式为将配置添加到 `flink-conf.yaml` 文件中。如果要在 S3 中使用 checkpoint，推荐使用 Presto S3 文件系统。
 
   - `flink-s3-fs-hadoop`，通过 *s3://* 和 *s3a://* 两种 scheme 使用, 基于 [Hadoop Project](https://hadoop.apache.org/)。
-  文件系统可以使用与 [Hadoop S3A 完全相同的配置方法](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#S3A)进行配置，即将配置添加到 `flink-conf.yaml` 文件中。它是唯一一个支持 [StreamingFileSink]({{ site.baseurl}}/zh/dev/connectors/streamfile_sink.html) 的文件系统。
-
+  本文件系统可以使用类似 [Hadoop S3A 的配置项](https://hadoop.apache.org/docs/stable/hadoop-aws/tools/hadoop-aws/index.html#S3A)进行配置，方式为将配置添加到 `flink-conf.yaml` 文件中。
+  
+     例如，Hadoop 有 `fs.s3a.connection.maximum` 的配置选项。 如果你想在 Flink 程序中改变该配置的值，你需要将配置 `s3.connection.maximum: xyz` 添加到 `flink-conf.yaml` 文件中。Flink 会内部将其转换成配置 `fs.s3a.connection.maximum`。 而无需通过 Hadoop 的XML 配置文件来传递参数。
```
Comment (klion26) [inline docs/ops/filesystems/s3.zh.md:66]:
```suggestion
     例如，Hadoop 有 `fs.s3a.connection.maximum` 的配置选项。 如果你想在 Flink 程序中改变该配置的值，你需要将配置 `s3.connection.maximum: xyz` 添加到 `flink-conf.yaml` 文件中。Flink 会内部将其转换成配置 `fs.s3a.connection.maximum`。 而无需通过 Hadoop 的 XML 配置文件来传递参数。
```

File: docs/ops/filesystems/s3.zh.md [hunk 5]
```diff
@@ -58,11 +58,15 @@ env.setStateBackend(new FsStateBackend("s3://<your-bucket>/<endpoint>"));
...
+  
+    另外，它是唯一支持 [StreamingFileSink]({{ site.baseurl}}/zh/dev/connectors/streamfile_sink.html) 的S3 文件系统。
```
Comment (klion26) [inline docs/ops/filesystems/s3.zh.md:68]:
```suggestion
    另外，它是唯一支持 [StreamingFileSink]({{ site.baseurl}}/zh/dev/connectors/streamfile_sink.html) 的 S3 文件系统。
```

File: docs/ops/filesystems/s3.zh.md (hunk 4 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: Thanks for your patience , this problem has been fixed

File: docs/ops/filesystems/s3.zh.md (hunk 5 above)
Comment (wydhcws) [reply docs/ops/filesystems/s3.zh.md]: Thanks for your patience , this problem has been fixed

**Comment by wydhcws** [comment]
Thanks for your patience and careful  . I've fixed the problem.  @klion26

**Comment by rmetzger** [comment]
Thanks a lot for your review. Merging.

**Pull request closed without merging**